feature/experimental/tunnel/otg_tests/tunnel_interface_based_resize_test/README.md

```diff
@@ -0,0 +1,64 @@
+# TUN-1.6: Tunnel End Point Resize - Interface Based GRE Tunnel
+
+## Summary
+
+*    Tunnel End Point Resize - Interface Based GRE Tunnel
+
+## Procedure
+
+*   Apply the config mentioned in Tunnel-1.1
+*   Send the traffic as mentioned in Tunnel-1.3 and Tunnel-1.4 with TP-1.1 and TP-1.2
+*   Modify the numbers of Tunnel interface being used 
+*   Reduce number of Tunnel interfaces(e.g. From 16 to 12):
+        - Incoming traffic on DUT-PORT1 should be load balanced to available Tunnel interfaces for encapsulation
+        - Incoming traffic flow should be equally distributed for Encapsulation(ECMP)
+*   If the static routes are used to forward traffic to tunnel, please disable or delete the static route in this test to simulate the reduction in available paths
+*   Increase number of Tunnel interfaces(e.g. From 16 to 20): 
+*   Incoming traffic on DUT-PORT1 should start using additional Tunnel Interfaces  for encapsulation
+*   Incoming traffic flow should be equally distributed for Encapsulation(ECMP)
+*   Increasing and decreasing the number of tunnel interfaces and related static route shouldn’t case traffic drops
+*   Verify the next hop counters for packet being diverted or sent for encapsulation
+*   Verify the tunnel interfaces counters to confirm the traffic encapsulation
+*   Verify the tunnel interfaces traffic/flow for equal distribution for optimal load balancing
+*   After decapsulation, traffic should be load balanced/hash to all available L3 ECMP or LAG or combination of both features
```

Comment:
```suggestion
*   After encapsulation, traffic should be load balanced/hash to all available L3 ECMP or LAG or combination of both features
```

Reply:
I am little confused here. Up to line#23 everything is about encap. All of a sudden after line#23 it is about Decap. where is this coming from? Who is sending the pre-encaped traffic